src/hb-ot-cff1-table.hh

```diff
@@ -1217,6 +1217,31 @@ struct cff1
       }
     }
 
+    hb_codepoint_t sid_to_glyph (hb_codepoint_t sid) const
+    {
+      if (charset != &Null(Charset))
```

Comment:
nullptr feels even better than nullpool for such cases :/

Reply:
Maybe so. This set up is not new though. There are about two dozens places like this.

Reply:
Understandable, feel free to remove some of them if you got some time to not have such surprises.